browse-scaler/src/__test__/resize.test.js

```diff
@@ -4,52 +4,60 @@ const { resizeImage, notFound } = require('../resize');
 
 describe('Image resizing', () => {
   test('Images are resized as expected', async () => {
-    const image = fs.readFileSync('stars.jpg');
+    const image = fs.readFileSync(`${__dirname}/stars.jpg`);
 
     const imageResponse = await resizeImage(image, 200, 200);
-    fs.writeFileSync('stars-resize.png', imageResponse, { encoding: 'base64' });
+    fs.writeFileSync(`${__dirname}/stars-resize.png`, imageResponse, {
+      encoding: 'base64'
+    });
 
-    const imgDimensions = await sharp('stars-resize.png').metadata();
+    const imgDimensions = await sharp(`${__dirname}stars-resize.png`).metadata();
```

Comment:
Looks like you're missing a `/` between `__dirname` and `stars-resize.png`.

Reply:
Ah I thought I got em' all! Fixed